backend/sdk/vnscope/classify.py

```
@@ -0,0 +1,371 @@
+        plt.show()
+
+    def analyze(self, symbols, number_of_levels, min_peak_distance=0.0):
+        """Run the full volume profile analysis pipeline.
+
+        Args:
+            df (pl.DataFrame, optional): Input DataFrame. Uses self.df if None.
+
+        Returns:
+            pl.DataFrame: DataFrame with [symbol, shape] and original columns.
+        """
+        from .core import profile, price
```

Ruff (0.11.9)

`.core.price` imported but unused

Remove unused import: `.core.price`

(F401)